### PR TITLE
Create cursedscrubdaddygang.json

### PR DESCRIPTION
**Pull request by QuantumMapleQC**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

image

## Website Link/Preview
image

### CODE REVIEW

File: domains/cursedscrubdaddygang.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "repo": "https://github.com/QuantumMapleQC/CursedScrubDaddyGang"
+        "username": "quantummapleqc"
+        "Discord": "herricksom"
+    },
+    "record": {
+        "CNAME": "quantummapleqc.github.io/CursedScrubDaddyGang/"
```
Comment (DEV-DIBSTER) [inline domains/cursedscrubdaddygang.json:8]: CNAME is invalid. CNAME should be a hostname without any additional components like protocols or file paths.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by QuantumMapleQC** [comment]
Github does not let me make a domain thats like

example.github.io

Instead it just gives me an used one along with the filepath

**Comment by DEV-DIBSTER** [comment]
@QuantumMapleQC You can set your hostname to be just `quantummapleqc.github.io`, which then you could access that URL at 
`https://cursedscrubdaddygang/CursedScrubDaddyGang/`

**Comment by QuantumMapleQC** [comment]
Thats strange cause github does not let me do this normally

File: domains/cursedscrubdaddygang.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "repo": "https://github.com/QuantumMapleQC/quantummapleqc.github.io"
+        "username": "quantummapleqc"
+        "Discord": "herricksom"
+    },
+    "record": {
+        "CNAME": "https://quantummapleqc.github.io/"
```
Comment (CuteDog5695) [inline domains/cursedscrubdaddygang.json:8]:
```suggestion
        "CNAME": "quantummapleqc.github.io"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems good to me

File: domains/cursedscrubdaddygang.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "repo": "https://github.com/QuantumMapleQC/quantummapleqc.github.io"
+        "username": "quantummapleqc"
+        "Discord": "herricksom"
```
Comment (CuteDog5695) [inline domains/cursedscrubdaddygang.json:5]:
```suggestion
        "repo": "https://github.com/QuantumMapleQC/quantummapleqc.github.io",
        "username": "quantummapleqc",
        "Discord": "herricksom"
```

**Review by iostpa — CHANGES_REQUESTED** [review]
NEVERMIND

**Review by iostpa — APPROVED** [review]
Now its fine

**Pull request merged**